### ISSUE TITLE
cavstool: add a new option to dump contents to the specified file

### ISSUE THREAD

**Issue by keqiaozhang** [zephyrproject-rtos/zephyr#45044]
**Is your enhancement proposal related to a problem? Please describe.**
We are switching to use the cavstool.py to dump the mailbox or DMA trace contents, we want to save the trace to a specified file for debugging propose.  It would be great if there was an option to dump the contents to a specified file.

**Describe the solution you'd like**
$ cavstool.py -l -o outfile

**Describe alternatives you've considered**
$ cavstool.py -l &> outfile

**Comment by marc-hb** [comment]
@keqiaozhang why is file redirection not enough?

**Comment by nashif** [comment]
please submit a PR, this is a small change to the script.

**Comment by marc-hb** [comment]
> @keqiaozhang why is stdout redirection > cavstool.log not enough?

There are various tricks to combine `sudo` + redirection, here's my favorite:

```shell
sudo cavstool.py ... | sudo tee [ -a ] log.txt > /dev/null
```

**Comment by keqiaozhang** [comment]
> ```shell
> | sudo tee [ -a ]
> ```

Yes, we can use stdout redirection. Now I'm using '&>' for debugging.
